weave

```diff
@@ -1285,6 +1286,13 @@ ipam_cidrs() {
         allocate)
             METHOD=POST
             CHECK_ALIVE="?check-alive=true"
+            # If we get an error here, we skip applying necessary steps for AWSVPC, because
+            # "attach" should work without the weave router running.
+            [ "$(call_weave GET /ipinfo/tracker)" != "awsvpc" ] || AWSVPC=1
```

Comment:
maybe move to a separate function, because there exist two instances of the same check and set.